app/io/pathfinder/routing/ClusterRouter.scala

```diff
@@ -11,7 +11,7 @@ import io.pathfinder.websockets.pushing.EventBusActor
 import io.pathfinder.websockets.{Events, ModelTypes}
 import play.Logger
 import play.api.Play
-import play.api.libs.json.{JsResultException, JsString, Json, JsArray, Reads, __, JsNumber, JsObject, Writes, JsValue}
+import play.api.libs.json.{JsResultException, JsString, JsArray, Reads, __, JsNumber, JsObject, Writes, JsValue}
```

Comment:
Yeah, it is for json parsing, you can also use variables named $ or %, and even < too if you want.